### PR TITLE
Auto-assign collaborator for review when not specified in PR body

### PR DESCRIPTION
**Pull request by magni-**
Diff minus #61: https://github.com/servo/highfive/compare/9403b90...715ccdd

Fixes: #40 

Rather than choose at random, this iterates over a list of collaborators. Is an `ini` file the best way to define these?

### CODE REVIEW

**Comment by KiChjang** [comment]
I'm kinda sad that my name isn't in the collaborators.ini file.

**Comment by KiChjang** [comment]
Here's the current list of reviewers: https://github.com/servo/saltfs/blob/master/homu/cfg.toml#L63

**Comment by magni-** [comment]
Sorry :( I copied what was in the welcome handler...

File: handlers/assign_reviewer/__init__.py [hunk 1]
```diff
@@ -1,15 +1,47 @@
...
         return None
     return match.group(1)
 
+# Return a collaborator's username if there is one in the config file. Otherwise, return None.
+def choose_reviewer(api, pull_number):
+    collaborators = get_collaborators(api)
+    if not collaborators:
+        return None
+
+    return collaborators[pull_number % len(collaborators)]
```
Comment (jdm) [inline handlers/assign_reviewer/__init__.py]: Ha, it's cute that the PR number is used to choose one.

**Comment by magni-** [comment]
Ugh what did I just do... past midnight here, I'll fix this tomorrow.

**Comment by jdm** [comment]
Ok! This looks like it's ready to merge, so I'll probably go ahead and cherry-pick the right commits. Thanks!

**Comment by magni-** [comment]
Ahh thanks! Should be just the last three

**Comment by jdm** [comment]
Merged as c58f22be34e78493a0260934c12bcf3ab86f24a1. Thanks!

**Pull request closed without merging**